projects/plugins/jetpack/modules/contact-form/js/dropdown.js

```diff
@@ -0,0 +1,23 @@
+jQuery( function ( $ ) {
+	$( document ).ready( function () {
+		initializeSelectMenu();
+
+		const observer = new MutationObserver( () => {
+			initializeSelectMenu();
+		} );
+
+		observer.observe( document.querySelector( 'body' ), {
```

Comment:
@ice9js do you think there's a better way to implement this observer?
I mean, without observing `body` changes.
Not sure if there are performance implications here.

Reply:
I think this is generally fine - the observer itself is fairly light but one thing to watch out for is that we're not needlessly regenerating everything anytime the DOM changes, which could be a lot.

But I see you've taken care of that with `generateStyleVariables` which only creates the 'probe' once per page load and I'd assume `.selectmenu()` must implement some similar conditions or it'd end up in an endless loop since it does mutate the DOM itself.